Assets/UX/Scripts/LocalizationManager.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Localization;
+using UnityEngine.Localization.Settings;
+using UnityEngine.Localization.Tables;
+using UnityEngine.ResourceManagement.AsyncOperations;
+
+public class LocalizationManager : MonoBehaviour
+{
+    public enum SupportedLanguages
+    {
+        ChineseSimplified,
+        English,
+        French,
+        German,
+        Hindi,
+        Italian,
+        Japanese,
+        Korean,
+        Portuguese,
+        Russian,
+        Spanish,
+        Tamil,
+        Telugu
+    }
+
+    public SupportedLanguages CurrentLocalizedLanguage;
+
+    const string k_ReasonTable = "Reasons";

```

Comment:
I recommend saving these keys as enums (4 bytes) instead of strings(2 bytes per character) and then use the GetName() method when using the key. Since the language switching will not occur very often, I would lean towards saving memory instead of the temporal allocation when creating the string.